lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -61,25 +60,24 @@ def __del__(self):
     def __call__(self, notification):
         if isinstance(notification, ufe.AttributeValueChanged):
             if notification.name() == UsdGeom.Tokens.xformOpOrder:
-                mel.eval("evalDeferred(\"AEbuildControls\");")
+                mel.eval("evalDeferred -low \"refreshEditorTemplates\";")
```

Comment:
I found this Maya command which does a AEbuildControls and hen refreshed the copied layouts. So this fixes a bug where a copied tab would not update correctly after adding a new transform op.